scripts/teamshow

```diff
@@ -110,16 +118,27 @@ class Teamshow(object):
                 info['ports'] = 'N/A'
             else:
                 for port in json_info['ports']:
-                    info['ports'] += port
-                    info['ports'] += '(S)' if json_info['ports'][port]['runner']['selected'] else '(D)'
-                    info['ports'] += ' '
+                    status = self.get_portchannel_member_status(team, port)
+                    selected = json_info["ports"][port]["runner"]["selected"]
+
+                    info["ports"] += port + "("
+                    info["ports"] += "S" if selected else "D"
+                    if status is None or (status == "enabled" and not selected) or (status == "disabled" and selected):
+                        info["ports"] += "*"
+                    info["ports"] += ") "
+
             self.summary[team_id] = info
 
     def display_summary(self):
         """
             Display the portchannel (team) summary.
         """
-        print "Flags: A - active, I - inactive, Up - up, Dw - Down, N/A - not available, S - selected, D - deselected"
+        print \
+"""\
```

Comment:
Lack of indentation here makes the code difficult to follow. Suggest the following syntax instead:
```
print("Flags: A - active, I - inactive, Up - up, Dw - Down, N/A - not available,\n"
      "       S - selected, D - deselected, * - not synced")

Reply:
resolved